tools/netsource.c

```diff
@@ -341,97 +373,78 @@ process (jack_nframes_t nframes, void *arg)
 
 
     if( reply_port )
-	input_fd = insockfd;
+        input_fd = insockfd;
     else
-	input_fd = outsockfd;
+        input_fd = outsockfd;
 
     // for latency == 0 we can poll.
-    if( (latency == 0) || (freewheeling!=0)  ) {
-	jack_time_t deadline = jack_get_time() + 1000000 * jack_get_buffer_size(client)/jack_get_sample_rate(client);
-	// Now loop until we get the right packet.
-	while(1) {
-		jack_nframes_t got_frame;
-#ifdef __JACK1__
```

Comment:
the `__JACK1__` and `__JACK2__` macros are being removed here, is this intended?

Reply:
Cherry-picked and adapted 7a8db80a703543b12097f9c32048b7784a69363c

Reply:
For posterity: I have removed that commit again, as it turns out that we confused one another with the assumption, that the functions in use were part of the JACK API whereas they are actually defined and implemented in common/netjack_packet.{c,h}.

Both jack1 and jack2 can be compiled just fine against the current implementation.